### PR TITLE
[docs-only] User Management through CS3 APIs ADR

### PR DESCRIPTION
**Pull request by refs**

### CODE REVIEW

File: docs/ocis/adr/0005-cs3-api-account-management.md [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+
+# 5. Account Management via CS3 API
+
+* Status: proposed
+* Deciders: @refs, @butonic, @micbar, @dragotin, @pmaier1
+* Date: 2021-04-12
+
+Technical Story: [Write only management API for User and Group resources](https://github.com/cs3org/cs3apis/pull/119)
+
+## Context and Problem Statementf
```
Comment (fschade) [inline docs/ocis/adr/0005-cs3-api-account-management.md:17]: Context and Problem Statement

File: docs/ocis/adr/0005-cs3-api-account-management.md [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+│                                                   │
+│ (1) solidify driver interfaces with write         │
+│                                                   │
+│ (2) an example of a driver could be GLAuth        │
+│ implementing the user management portion of the   │
+│ GraphAPI                                          │
+└───────────────────────────────────────────────────┘
+```
+
+This flow allows Reva and oCIS Proxy to runs any middleware logic in order to validate a request. The communication between the proposed Admin api (CS3 API messages) with the IDM (GLAuth) are specific to the _drivers_.
```
Comment (micbar) [inline docs/ocis/adr/0005-cs3-api-account-management.md:211]:
```suggestion
This flow allows Reva and oCIS Proxy to run any middleware logic in order to validate a request. The communication between the proposed Admin api (CS3 API messages) and the IDM (GLAuth) are specific to the _drivers_.
```

File: docs/ocis/adr/0005-cs3-api-account-management.md [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+│      │          │                  │          │ │ │
+│ │    └──────────┘           │  │   └──────────┘   │
+│                                                 │ │
+│ │                           │  │                  │
+│  ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─ ─    ─ ─ ─ ─ ─ ─ ─ ─ ┘ │
+│                                                   │
+│                                                   │
+│                                                   │
+│                                                   │
+│ (1) solidify driver interfaces with write         │
```
Comment (micbar) [inline docs/ocis/adr/0005-cs3-api-account-management.md:203]: how will it be solidified?

**Review by micbar — COMMENTED** [review]
LGTM, minor wordings and typos

File: docs/ocis/adr/0005-cs3-api-account-management.md (hunk 3 above)
Comment (refs) [reply docs/ocis/adr/0005-cs3-api-account-management.md:203]: leftover, good catch

**Review by micbar — APPROVED** [review]
LGTM

**Pull request merged**